### ISSUE TITLE
Light brightness slider can't be dragged on iOS app.

### ISSUE THREAD

**Issue by yosoyez** [UI-Lovelace-Minimalist/UI#590]
When using a light button card with a brightness adjustment slighter, it doesn't seem like it supports placing my finger and dragging it to the position I would like, I have to press the different spots which makes it hard to get an accurate position. Opening the light 'more info' dialog and dragging the built in slider works just fine. It would be great to have a smooth light slider on my phone.

**Comment by superhaha668** [comment]
the same to me, I am very eager to fix this bug, who can help me?

**Comment by superhaha668** [comment]
Have you solved this problem

**Comment by devonhumes** [comment]
The issue is due to iOS not correctly seeing the slider control without being able to touch it. The main issue is that the t[humb control width is set to 0px](https://github.com/UI-Lovelace-Minimalist/UI/blob/e1fae90c94e173cef1a437b17d0d9b33cc9fb192/custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml#L287).

With some initial testing, I was able to fix this by "hiding" the thumb control. Instead of setting the thumb control width to 0px I set the thumb control color to match the primary color and that made it disappear visually while still being present.

```yaml
        mainSliderColor: >
          [[[
              var color = entity.attributes.rgb_color;
              if (entity.state == "unavailable"){
                 return "rgba(var(--color-grey),1)";
              }
              else if (color && variables.ulm_card_light_enable_color){
                 return "rgba(" + color + ",1)";
              }
              else{
                return "rgba(var(--color-yellow),1)";
              }
          ]]]
        secondarySliderColor: >
          [[[
              var color = entity.attributes.rgb_color;
              if (entity.state == "unavailable"){
                 return "rgba(var(--color-grey),0.2)";
              }
              else if (color && variables.ulm_card_light_enable_color){
                 return "rgba(" + color + ",0.2)";
              }
              else{
                return "rgba(var(--color-yellow),0.2)";
              }
          ]]]
        thumbColor: >
          [[[
              var color = entity.attributes.rgb_color;
              if (entity.state == "unavailable"){
                 return "rgba(var(--color-grey),1)";
              }
              else if (color && variables.ulm_card_light_enable_color){
                 return "rgba(" + color + ",1)";
              }
              else{
                return "rgba(var(--color-yellow),1)";
              }
          ]]]
```

**Comment by devonhumes** [comment]
For reference, here's an issue with the legacy card repository. https://github.com/AnthonMS/ha-slider-card/issues/21#issuecomment-900176451

Also here's a post from the HA community talking about this issue. https://community.home-assistant.io/t/lovelace-ui-minimalist/322687/1044

**Comment by digaus** [comment]
@devonhumes 
that kinda works but is still not a great solution since you have to start dragging at the thumb control :/

**Comment by basbruss** [comment]
closed with #756
